policy/modules/services/matrixd.te

```diff
@@ -77,6 +77,7 @@ corecmd_exec_shell(matrixd_t)
 
 corenet_tcp_bind_generic_node(matrixd_t)
 corenet_tcp_bind_http_port(matrixd_t)
+corenet_tcp_bind_all_unreserved_ports(matrixd_t)
```

Comment:
What's the purpose for this?

Reply:
Allow me to double-check, but I recall I needed this access or else images and other media would fail to send over federation.

Reply:
OK this turned out to be because I am using Synapse workers in my setup, and some workers need to bind to a different HTTP port and may (optionally) expose Prometheus metrics on other ports. I'll instead make this a tunable for the `matrixd` policy.